src/main/res/values/strings.xml

```diff
@@ -791,7 +791,7 @@
 	<string name="submit_post_url_hint">URL</string>
 
 	<!-- 2016-06-24 -->
-	<string name="save_image_permission_denied">Permission denied: could not save image.</string>
+	<string name="save_image_permission_denied">Permission denied: Could not save image.</string>
```

Comment:
Capital not needed after a colon

Reply:
(applies to all changes of this type)

Reply:
Except in most other languages, reducing errors in translation.